components/Badge/Badge.css

```diff
@@ -7,7 +7,7 @@
     height: 93.75rem;
 }
 .component-library-components-container{
-    margin: 6.25rem auto 2.25rem 5.875rem; /* margin-top  margin-right margin-bottom margin-left  */
+    margin: 6.25rem auto 2.25rem 21.875rem; /* margin-top  margin-right margin-bottom margin-left  */
```

Comment:
can you remove the commented css

Reply:
yeaa i have done it thanks for reviewing